runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/stores/GlusterFileStore.java

```diff
@@ -70,7 +73,8 @@ public Block createBlock(final String blockId) {
     final String filePath = DataUtil.blockIdToFilePath(blockId, fileDirectory);
     final RemoteFileMetadata metadata =
       RemoteFileMetadata.create(DataUtil.blockIdToMetaFilePath(blockId, fileDirectory));
-    return new FileBlock<>(blockId, serializer, filePath, metadata);
+    final MemoryPoolAssigner memoryPoolAssigner = getMemoryPoolAssigner();
```

Comment:
Very very minor suggestion (here and other FileStores): call getMemoryPoolAssigner() directly in L77 to reduce the number of variables?

return new FileBlock(..., getMemoryPoolAssigner())